core/templates/dev/head/components/RatingDisplayDirective.js

```diff
@@ -16,88 +16,92 @@
  * @fileoverview Directive for displaying summary rating information.
  */
 
-oppia.directive('ratingDisplay', [function() {
-  return {
-    // This will display a star-rating based on the given data. The attributes
-    // passed in are as follows:
-    //  - isEditable: true or false; whether the rating is user-editable.
-    //  - onEdit: should be supplied iff isEditable is true, and be a function
-    //    that will be supplied with the new rating when the rating is changed.
-    //  - ratingValue: an integer 1-5 giving the rating
-    restrict: 'E',
-    scope: {
-      isEditable: '=',
-      onEdit: '=',
-      ratingValue: '='
-    },
-    templateUrl: 'components/ratingSummary',
-    link: function(scope, element) {
-      // This is needed in order for the scope to be retrievable during Karma
-      // unit testing. See http://stackoverflow.com/a/29833832 for more
-      // details.
-      element[0].getControllerScope = function() {
-        return scope;
-      };
-    },
-    controller: ['$scope', function($scope) {
-      var POSSIBLE_RATINGS = [1, 2, 3, 4, 5];
-      $scope.stars = POSSIBLE_RATINGS.map(function(starValue) {
-        return {
-          cssClass: 'fa-star-o',
-          value: starValue
+oppia.directive('ratingDisplay', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      // This will display a star-rating based on the given data. The attributes
+      // passed in are as follows:
+      //  - isEditable: true or false; whether the rating is user-editable.
+      //  - onEdit: should be supplied iff isEditable is true, and be a function
+      //    that will be supplied with the new rating when the rating is
+      //    changed.
+      //  - ratingValue: an integer 1-5 giving the rating
+      restrict: 'E',
+      scope: {
+        isEditable: '=',
+        onEdit: '=',
+        ratingValue: '='
+      },
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/components/rating_display.html'),
```

Comment:
Shouldn't this be rating_display_directive.html?

Reply:
the name of file is rating_display.html. I agree that it is not in accordance with convention (this is one reason that this file was left during whole refactor as this doesn't have directive in its name).

Reply:
Actually, I think if you fix the name, that will fix the issue. I remember running into this issue before -- only some file names will be transferred to the production build.